flink/v1.16/flink/src/test/java/org/apache/iceberg/flink/TestHelpers.java

```diff
@@ -295,6 +299,161 @@ private static void assertEquals(
     }
   }
 
+  public static void assertEqualsSafe(
+      Schema schema, List<GenericData.Record> recs, List<Row> rows) {
```

Comment:
why do we need the helper methods with Avro generic record?

Reply:
The metadata files are stored in avro format, and when we read them directly, we get the Avro generic record.

Reply:
ok. these new `assertEqualsSafe` methods seem only used inside this class. should they be private?

Reply:
`assertEqualsSafe(Types.StructType struct, GenericData.Record rec, Row row)` has beed changed to private. The other two are not changed because they are used in `TestFlinkMetaDataTable`.
Thx.